lib/scrabble.rb

```diff
@@ -1,7 +1,17 @@
 class Scrabble
 
   def score(word)
-    1
+    point =""
```

Comment:
Careful with this. Makes the test pass, and will give us an accurate score for a single letter, but starts to fall apart when you try to score words.

Would like to see some additional tests to check implementation for words, which would've started you down a path of thinking of how to make that work.

At a high level, what you would want to do in order to score a full word would be to:

1) split the word into individual letters,
2) get a score for each letter, and
3) sum the scores.

Take some time to implement this functionality to practice.